modules/json/src/json-table-loader.js

```diff
@@ -0,0 +1,105 @@
+/* global Buffer */
+import {AsyncQueue} from '@loaders.gl/experimental';
+import {TableBatchBuilder, RowTableBatch} from '@loaders.gl/experimental/categories/table';
+import JSONParser from './lib/json-parser';
+
+export default {
```

Comment:
I'd prefer to merge this with the `JSONLoader`. This is a new module and it doesn't look like they have any conflicts...

Reply:
> I'd prefer to merge this with the `JSONLoader`. This is a new module and it doesn't look like they have any conflicts 

I am not sure that makes sense. Two reasons to keep them separate:

- The `JSONTableLoader` is not a loader for generic JSON objects (unstructured "tree" of data). This is a loader for JSON containing a "top-level" array of JSON objects with identical, relatively flat structure. We still need a generic `JSONLoader`.

- The JSONLoader is 5 lines and extremely lightweight. This loader will be at least 1000 lines, and likely also pull in table batch library from the tables category.